tests/test_tomography.py

```diff
@@ -172,3 +173,19 @@ def test_matrix_basis_n_decomp():
         matrix_rebuilt += mu[idx]*basis_matrices
 
     assert np.allclose(matrix, matrix_rebuilt)
+
+
+def test_processor_circuit_configurator():
```

Comment:
Ah sorry, still one thing: you've got a conflict in this file vs the target branch.
And actually, this conflict relies around this new test which shouldn't be in this file, I guess. It's not a tomography test properly speaking. Could you move this test out of this file, and it should automatically fix the git conflict?